packages/sqlite_async/lib/src/common/sqlite_database.dart

```diff
@@ -103,4 +103,9 @@ abstract class SqliteDatabase
   factory SqliteDatabase.singleConnection(SqliteConnection connection) {
     return SingleConnectionDatabase(connection);
   }
+
+  /// Returns a list of all the connections (read and write) managed by this database.
+  /// This can be useful to run the same statement on all connections. For instance,
+  /// ATTACHing a database, that is expected to be available in all connections.
+  List<SqliteConnection> getAllConnections();
```

Comment:
If we go with this approach, let's make this a getter :)

```suggestion
  List<SqliteConnection> get allConnections;
```